cardinal/ecs/gamestate/store/iface.go

```diff
@@ -60,9 +60,9 @@ type TickStorage interface {
 	Recover(txs []message.Message) (*txpool.TxQueue, error)
 }
 
-// IManager represents all the methods required to track Component, Entity, and Archetype information
+// IGameStateManager represents all the methods required to track Component, Entity, and Archetype information
```

Comment:
I suggesting merging this package with the gamestate package one level above. So there would no longer be a cardinal/ecs/gamestate/store package. Just cardinal/ecs/gamestate.